### ISSUE TITLE
 rpn_cls_prob overwritten by rpn_bbox_pred

### ISSUE THREAD

**Issue by jugalsheth92** [roytseng-tw/Detectron.pytorch#10]
@roytseng-tw Nice work BTW, 

https://github.com/roytseng-tw/Detectron.pytorch/blob/master/lib/modeling/rpn_heads.py#L106-L109
*rpn_cls_prob* values are overwritten by *rpn_bbox_pred*, 

Is this a bug or its intentional?

**Comment by roytseng-tw** [comment]
Thank you, it's a bug for the case of using softmax instead of sigmoid as the activation function for the rpn class prediction.
I fixed it in eab3338fb69605029e29a5646e1521928019b2fb.
BTW, the default activation function is sigmoid.
